go-controller/pkg/config/config.go

```diff
@@ -16,15 +17,31 @@ import (
 // The following are global config parameters that other modules may access directly
 var (
 	// Default holds parsed config file parameters and command-line overrides
-	Default DefaultConfig
+	Default = DefaultConfig{
+		MTU:           1400,
+		ConntrackZone: 64000,
+	}
+
 	// Logging holds logging-related parsed config file parameters and command-line overrides
-	Logging LoggingConfig
+	Logging = LoggingConfig{
+		File:  "/var/log/ovn-kubernetes.log",
+		Level: 4,
+	}
+
 	// CNI holds CNI-related parsed config file parameters and command-line overrides
-	CNI CNIConfig
+	CNI = CNIConfig{
+		ConfDir: "/etc/cni/net.d",
+		Plugin:  "ovn-k8s-cni-overlay",
+	}
+
 	// Kubernetes holds Kubernetes-related parsed config file parameters and command-line overrides
-	Kubernetes KubernetesConfig
+	Kubernetes = KubernetesConfig{
+		APIServer: "http://localhost:8443",
```

Comment:
That should be the default value, I think it should be *http**s**://localhost:8443*

Reply:
@shettyg @alinbalutoiu yeah, I mean 8080 because we want the unauthenticated port, since by default we don't have/know the certificates necessary for HTTPS.